### ISSUE TITLE
Can't compile the extension

### ISSUE THREAD

**Issue by fixdude** [YoYoGames/GMEXT-FMOD#6]
### Description

I have followed every instruction on the getting started page correctly and imported the v0.0.2 yymps, but whenever i try to compile i get a LoadLibraryW error with code 126, on every function of the extension (i believe) and it does not work at all. I have read that the error has something to do with the runtime not matching the extension version, but both should be x64. (I am using the latest gamemaker version, along with the latest version of this extension as of writing this)

[compiler_log.txt](https://github.com/YoYoGames/GMEXT-FMOD/files/14464982/compiler_log.txt)

YYZ Link (It seems github doesn't support this format, so i hope i'm doing it right by uploading it to google drive and linking it here)
https://drive.google.com/file/d/1F--KxEPJsnKLntclOiNlmrTiImGkm9HN/view?usp=sharing

### Steps To Reproduce

1. Start GameMaker
2. Import the extension and follow the instructions
3. Try to compile
4. See the issue (?)

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

Monthly

### Which platform(s) are you seeing the problem on?

Windows

### Compiler Log File

- [X] I have attached my compiler log in a text file

### Sample .yyz File

- [X] I have attached a small sample project

**Comment by DiasFranciscoA** [comment]
@fixdude, we appreciate you bringing this to our attention. We've recently updated the package, so we kindly ask you to give it another try. Initially, the libraries were compiled without using Release mode, which may have caused issues if Visual Studio wasn't installed on your system, as it's needed for debug compilations. We've addressed this oversight, and the updated package should now function correctly on your machine.
